index.js

```diff
@@ -1,15 +1,32 @@
-var FluxConstant = function FluxConstant(name) {
+var FluxConstant = function FluxConstant(name, id) {
 
-    this.name = name;
+    this.id = String(id || (FluxConstant.lastId += 1));
```

Comment:
If you're trying to keep the state of `FluxConstant.lastId` you can set it manually. I do this in the tests here: https://github.com/jedireza/flux-constant/blob/72f3ff898613f7d1937b7018315e95f2447e93da/test/index.js#L48